### ISSUE TITLE
Fix python unit tests

### ISSUE THREAD

**Issue by oktaal** [UUDigitalHumanitieslab/I-analyzer#99]
Following is thrown running the tests in `feature/redesign-search`.

```bash
============================= test session starts ==============================
platform linux -- Python 3.6.3, pytest-3.0.6, py-1.4.33, pluggy-0.4.0
rootdir: /home/sheean/src/I-analyzer/api, inifile: 
collected 4 items

ianalyzer/tests/test_corpusimport.py FF.
ianalyzer/tests/test_times.py F

=================================== FAILURES ===================================
________________________________ test_key_error ________________________________

monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f1d1d19b978>
capsys = <_pytest.capture.CaptureFixture object at 0x7f1d1d19b9b0>

    def test_key_error(monkeypatch,capsys):
    	''' Verify that exception is correctly raised
    	- in case the config.CORPORA variable is empty
    	'''
    	with pytest.raises(KeyError) as e:
    		monkeypatch.setattr(config, 'CORPORA', {})
>   		reload(corpora)
E     Failed: DID NOT RAISE <class 'KeyError'>

ianalyzer/tests/test_corpusimport.py:23: Failed
______________________________ test_import_error _______________________________

monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f1d1d15b390>
capsys = <_pytest.capture.CaptureFixture object at 0x7f1d21cd0278>

    def test_import_error(monkeypatch,capsys):
    	''' Verify that exceptions is correctly raised
    	- in case the file path in config.CORPORA is faulty
    	'''
    	with pytest.raises(FileNotFoundError) as e:
    		monkeypatch.setattr(config, 'CORPUS', 'times')
    		monkeypatch.setattr(
    			config, 'CORPORA', {'times': '/ianalyzer/corpora/tmes.py'}
    		)
>   		reload(corpora)
E     Failed: DID NOT RAISE <class 'FileNotFoundError'>

ianalyzer/tests/test_corpusimport.py:37: Failed
______________________________ test_times_source _______________________________

    def test_times_source():
        '''
        Verify that times source files are read correctly.
        '''
        # initialize the corpora module within the testing context
        reload(corpora)
    
        print(dirname(__file__), corpora.corpus_obj.data_directory)
    
        # Assert that indeed we are drawing sources from the testing folder
>       assert dirname(__file__) in corpora.corpus_obj.data_directory
E       assert '/home/sheean/src/I-analyzer/api/ianalyzer/tests' in '/home/sheean/ownCloud/Shared/DigitalHumanitiesLabatUU/I-Analyzer'
E        +  where '/home/sheean/src/I-analyzer/api/ianalyzer/tests' = dirname('/home/sheean/src/I-analyzer/api/ianalyzer/tests/test_times.py')
E        +  and   '/home/sheean/ownCloud/Shared/DigitalHumanitiesLabatUU/I-Analyzer' = <dutchbanking.DutchBanking object at 0x7f1d1d17ae48>.data_directory
E        +    where <dutchbanking.DutchBanking object at 0x7f1d1d17ae48> = corpora.corpus_obj

ianalyzer/tests/test_times.py:37: AssertionError
----------------------------- Captured stdout call -----------------------------
/home/sheean/src/I-analyzer/api/ianalyzer/tests /home/sheean/ownCloud/Shared/DigitalHumanitiesLabatUU/I-Analyzer
====================== 3 failed, 1 passed in 0.13 seconds ======================
```
